web/src/timePosted.js

```diff
@@ -1,15 +1,13 @@
 export default (postedAtSeconds) => {
+  const minute = 60, hour = 3600, day = 86400, week = 604800;
   const currentTimeAtSeconds = new Date().getTime() / 1000;
-  const deltaAtSeconds = currentTimeAtSeconds - postedAtSeconds
-  if (deltaAtSeconds < 60) {
-    return `${Math.floor(deltaAtSeconds)}s`
-  } else if (deltaAtSeconds < 3600) {
-    return `${Math.floor(deltaAtSeconds / 60)}m`
-  } else if (deltaAtSeconds < 3600 * 24) {
-    return `${Math.floor(deltaAtSeconds / 3600)}h`
-  } else if (deltaAtSeconds < 3600 * 24 * 7) {
-    return `${Math.floor(deltaAtSeconds / (3600 * 24))}d`
-  } else {
-    return new Date(postedAtSeconds * 1000).toISOString().split('T')[0];
-  }
+  const deltaAtSeconds = currentTimeAtSeconds - postedAtSeconds;
+  const d = deltaAtSeconds;
+  return (
```

Comment:
I actually think the original else if's are more readable, but that's up to personal preference